### DISCUSSION TITLE
Would love buttons 'unhide all entities' and 'enable all entities'

### DISCUSSION THREAD

**Discussion by MrMxyzptlk** [dwainscheeren/dwains-lovelace-dashboard#658] [Ideas]
Would love buttons 'unhide all entities' and 'enable all entities', on the area entities pages along with the 'hide all entities' and 'disable all entities'.

Thank you.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
